BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+"""Root configuration for Bazel build tool.
+"""
+
+# Allow the tsconfig.json file to be referenced by subpackages
+exports_files(["tsconfig.json"])
+
+# Provides the npm-installed dependencies to Bazel actions.
+filegroup(
+    name = "node_modules",
+    srcs = glob(
+        ["node_modules/**/*"],
+        # Exclude directories that commonly contain filenames which are
+        # illegal bazel labels
+        exclude = [
+            # e.g. node_modules/adm-zip/test/assets/attributes_test/New folder/hidden.txt
+            "node_modules/**/test/**",
+            # e.g. node_modules/xpath/docs/function resolvers.md
+            "node_modules/**/docs/**",
+        ],
+    ),
+    # Allow this targets to be referenced by anyone.
+    # Other repositories running nodejs binaries will need to
+    # reference our installed node_modules directory.
+    visibility = ["//visibility:public"],
+)
+
+# Temporary target to allow `bazel test ...`
+# TODO(alexeagle): remove as soon as there is any other test in the repo
+genrule(
+    name = "true",
+    outs = ["true.sh"],
+    cmd = "echo true > $@",
+)
+
+sh_test(
```

Comment:
What exactly does this do?

Reply:
it's just a target which can be run with `bazel test`, and the shell script it executes just runs the posix command "true" which is the most trivial test I can come up with. I'll remove it and add a real test instead.

Reply:
Actually, I'll just remove the bazel test command from circleci